### ISSUE TITLE
[AXE-CORE] STEM Standalone - ID attribute value MUST be unique

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#12325]
# [508-defect-1 :exclamation: Launchblocker](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-1)

**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

The [axe-core browser plugin](https://deque.com/axe) is reporting an ID violation. We have two `div[id="save-in-progress-intro"]` containers on the page. This will cause a break in the CI/CD pipeline and needs to be remediated before launch.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Acceptance Criteria

- [ ] Existing functionality and layout is maintained
- [ ] axe issue "ID attribute must be unique" no longer appears
- [ ] No new axe errors appear
- [ ] Page should have e2e `axeCheck()` tests for the `[div.main]` container

## Environment

* https://staging.va.gov/education/apply-for-education-benefits/application/10203/introduction
* Logged out user

## WCAG or Vendor Guidance (optional)

* [Parsing: Understanding SC 4.1.1](https://www.w3.org/TR/UNDERSTANDING-WCAG20/ensure-compat-parses.html)

## Screenshots or Trace Logs

<img width="693" alt="Screen Shot 2020-08-10 at 4 27 08 PM" src="https://user-images.githubusercontent.com/934879/89833645-027a5300-db27-11ea-8f75-10c830dd611d.png">

---
<img width="647" alt="Screen Shot 2020-08-10 at 4 27 14 PM" src="https://user-images.githubusercontent.com/934879/89833650-04dcad00-db27-11ea-80a8-58a79a0d192a.png">

**Comment by zurbergram** [comment]
https://github.com/department-of-veterans-affairs/vets-website/pull/13817

**Comment by sonntag-adam** [comment]
![Screen Shot 2020-08-17 at 1.04.48 PM.png](https://images.zenhubusercontent.com/5c98df9c534e09000148f8d1/954f06a7-f309-4904-a16b-94d0f1d55939)

**Comment by sonntag-adam** [comment]
@1Copenut the fix is now in staging.

**Comment by 1Copenut** [comment]
@sonntag-adam I concur. The axe scan reported 0 violations on https://staging.va.gov/education/other-va-education-benefits/stem-scholarship/apply-for-scholarship-form-22-10203/introduction. Moving to close.
